app/Http/Controllers/Portal/Contact/ContactController.php

```diff
@@ -655,4 +695,161 @@ public function relatedAdministrations(Contact $contact)
         return AdministrationResource::collection($administrations);
     }
 
+    /**
+     * @param $project
+     * @param Contact $contact
+     * @param $projects
+     * @param int $key
+     */
+    protected function setContactProjectIndicators($project, Contact $contact, $projects, int $key)
+    {
+        $project->isSceOrPcrProject = $project->projectType->code_ref === 'postalcode_link_capital' || $project->is_sce_project;
+        $project->hasParticipation = false;
+        $project->allowChangeParticipation = false;
+        $project->allowPayMollie = false;
+        $project->econobisPaymentLink = '';
+        $project->allowRegisterToProject = false;
+        $project->textNotAllowedRegisterToProject = '';
+        $project->participationsOptioned = 0;
+        $project->amountOptioned = 0;
+        $project->powerKwhConsumption = 0;
+
+        $previousParticipantProject = $contact->participations()->where('project_id', $project->id)->first();
+        // Is there allready a participation for this contact/project ?
+        if ($previousParticipantProject) {
+            $project->hasParticipation = true;
+            $project->participationsOptioned = $previousParticipantProject->participations_optioned;
+            $project->amountOptioned = $previousParticipantProject->amount_optioned;
+            $project->powerKwhConsumption = $previousParticipantProject->power_kwh_consumption;
+            $previousMutation = optional(optional($previousParticipantProject)->mutations())->first(); // Pakken de eerste mutatie, er zou er altijd maar een moeten zijn op dit moment.
+
+            // Is mollie is used and there was a first mutation with status option with is paid by mollie false, then:
```

Comment:
Comment klopt niet helemaal en kun je niet beter deze syntax gebruiken voor multi-line comments  /* */

Reply:
Changed comment. (maar in PHP storm komt ie default met single-line comments, kan niet garanderen dus dat ik altijd multi-line comments gebruik).